naughty/rhel-9/2840-kdump-no-memory

```diff
@@ -0,0 +1 @@
+b.wait_in_text("#app", "Service is running")
```

Comment:
But again, this is way too generic. We talk to a lot more than one service, and more than one page's top level element is called `#app`.